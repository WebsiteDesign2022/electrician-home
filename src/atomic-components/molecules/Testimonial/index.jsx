import React from 'react';

import './index.scss';

const Testimonial = ({ description, name }) => (
    <div className="col d-flex align-items-start px-6 gx-6 bg-white testimonial">
        <div className="m-12">
            <p align="left" className="testimonial-content">{description}</p>
            <h6 align="left">{name}</h6>
        </div>
    </div>
);

export default Testimonial;