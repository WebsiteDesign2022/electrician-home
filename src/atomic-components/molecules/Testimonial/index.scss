.testimonial {
    padding: 64px;
    margin: 32px;
    background-color: white;
}

.testimonial-content {
    color: black;
}