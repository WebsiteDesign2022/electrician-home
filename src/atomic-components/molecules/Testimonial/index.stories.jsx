import React from 'react';

import Testimonial from '.';

export default {
  title: 'Electrician-Home/molecules/Testimonial',
  component: Testimonial,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <Testimonial {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
