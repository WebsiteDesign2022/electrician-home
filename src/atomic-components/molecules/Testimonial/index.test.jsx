import TestRenderer from 'react-test-renderer';

import Testimonial from '.';

describe("Testimonial", () => {
    test("renders an empty Testimonial", () => {
        const testimonial = TestRenderer.create(
            <Testimonial/>
        );

        expect(testimonial.toJSON()).toMatchSnapshot();
    });

    test("renders an empty Testimonial", () => {
        const testimonial = TestRenderer.create(
            <Testimonial name="testimonial-name" description="testimonial-description"/>
        );

        expect(testimonial.toJSON()).toMatchSnapshot();
    });
});
