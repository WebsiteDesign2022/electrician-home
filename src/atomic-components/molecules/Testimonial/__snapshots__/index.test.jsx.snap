// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial renders an empty Testimonial 1`] = `
<div
  className="col d-flex align-items-start px-6 gx-6 bg-white testimonial"
>
  <div
    className="m-12"
  >
    <p
      align="left"
      className="testimonial-content"
    />
    <h6
      align="left"
    />
  </div>
</div>
`;

exports[`Testimonial renders an empty Testimonial 2`] = `
<div
  className="col d-flex align-items-start px-6 gx-6 bg-white testimonial"
>
  <div
    className="m-12"
  >
    <p
      align="left"
      className="testimonial-content"
    >
      testimonial-description
    </p>
    <h6
      align="left"
    >
      testimonial-name
    </h6>
  </div>
</div>
`;
